MyConfig.h

```diff
@@ -442,7 +442,7 @@
  * - RF24_PA_MAX = 0dBm
  */
 #ifndef MY_RF24_PA_LEVEL
-#define MY_RF24_PA_LEVEL (RF24_PA_HIGH)
```

Comment:
I'd prefer if we didn't change the default PA level in this PR, since it is not related to the change.